components/develop/modules/ROOT/partials/deployment-scripts.adoc

```diff
@@ -44,59 +68,77 @@ dotenv.config();
-```
+[IMPORTANT]
+Certain plugins (e.g., hardhat-deploy ) can check previous transactions. If you are not connected to an Archive Node please use flags to disable the `boolean` for previous transactions check (for hardhat-deploy - skipIfAlreadyDeployed)
 
-**VERIFYING**
 
-Run the fowllowing command after updating it with the contract address returned from `hardhat run` and adding any simple constructors (if no constructors, nothing is needed- can be omitted)
+1. You will need to https://hardhat.org/hardhat-runner/docs/advanced/scripts#writing-scripts-with-hardhat[write or reuse a deployment script] in order deploy your smart contracts to SKALE with hardhat
+2. You can now use the flag `--networks skale` with your deployment scripts.
++
+*Example deployment command:*
+
+ npx hardhat run ./scripts/<<your script>> --network skale
 
-```shell
-hardhat verify <<contract address>> --network skale "add any simple contructors"
-```
-After a few moments, this will return a link to the Blockscout verified contract if successful. 
 
 [NOTE]
-For more info about conifgurations or adding complex contructors please visit: https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan
+After a successful deployment, the returned address will be used in the verification command. Save it before clearing your terminal!  
+ 
+ 
+=== VERIFYING
 
+1. Before verifying with hardhat CLI make sure you have installed the necessary dependencies:
++
+[source, shell]
+----
+npm install --save-dev "@ethersproject/providers@^5.4.7" "@nomicfoundation/hardhat-network-helpers@^1.0.0" "@nomicfoundation/hardhat-chai-matchers@^1.0.0" "@nomiclabs/hardhat-ethers@^2.0.0" "@types/chai@^4.2.0" "@types/mocha@^9.1.0" "@typechain/ethers-v5@^10.1.0" "@typechain/hardhat@^6.1.2" "chai@^4.2.0" "ethers@^5.4.7" "hardhat-gas-reporter@^1.0.8" "solidity-coverage@^0.7.21" "ts-node@>=8.0.0" "typechain@^8.1.0" "typescript@>=4.5.0" "dotenv@^16.0.1" "@openzeppelin/contracts@^4.7.0" "@nomicfoundation/hardhat-toolbox@^1.0.2" "@nomiclabs/hardhat-etherscan@^3.1.5"
+----
++
+2. Copy the address returned from deploying, and run the verification command
++
+[source, shell]
+npx hardhat verify <<contract address>> --network skale "add any simple constructor"
++
+[NOTE]
+====
+* If you have no constructors, the "add constructor" can be omitted
+* See https://hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-verify#complex-arguments[Hardhat Docs] for handling complex constructors (multiple type constructors)
+====
 
+After a few moments, this will return a link to the Blockscout verified contract if successful.
 --
 
 Truffle::
```

Comment:
Possible formatting issue here. Under the preview links for this PR the two tabs (truffle + hardhat) are not actually tabs. Request formatting fix